packages/salesforcedx-vscode-apex/test/vscode-integration/javaConfiguration.test.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// tslint:disable:no-unused-expression
```

Comment:
So I dug into this one a bit, and we do need it because of the `expect` blocks. It looks like it's possible to add a [plugin](https://www.npmjs.com/package/tslint-no-unused-expression-chai ) that would handle the chai files for us, but I wasn't sure we wanted to add another plugin. We could also create tslint configurations for every test directory, specifically to turn off the no-unused-expression linter, but that also felt a little heavy-handed for this PR. 🙂